WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -176,6 +176,11 @@ + (NSArray *)routes
   NSString *elementUUID = request.parameters[@"uuid"];
   XCUIElement *element = [elementCache elementForUUID:elementUUID];
   NSError *error = nil;
+  
+  //scrollelement to visiable but ignore errros happen during scroll
+  [element fb_scrollToVisibleWithError:&error];
```

Comment:
I think we definitely should check for error and implement it like this:
```
if(![element fb_scrollToVisibleWithError:&error]) {
   return FBResponseWithError(error);
}
```
If it fails on success it sounds like a bug and I'd rather fix it instead of hiding the problem ;).